### ISSUE TITLE
Can "averager" average over the time axis (and other averager questions) 

### ISSUE THREAD

**Issue by jypeter** [CDAT/cdat.github.io#197]
I have used `cdutil.averager` the last 2 days, and checked some of its documentation, because this is one of the (many) important (and specific) functions of CDAT, and I have several questions/remarks below

- `averager` used to be in cdutil, and the [documentation](https://cdat.llnl.gov/documentation/utilities/utilities.html) still places it in the cdutil documentation. But I have checked a bit after noticing that it was available in both _cdutil_ and _genutil_, and found out that it is coded in genutil. So should it also be moved to genutil in the documentation?

- I have used averager to average all the **time steps** and all the longitudes of a variable with `cdutil.averager(data_diff, axis='tx', weights=['weighted', 'weighted'])`, after calling `cdutil.setTimeBoundsMonthly` to make sure I had correct time bonds. Is the temporal average weighted correctly, even-though the [TOC](https://cdat.llnl.gov/documentation/utilities/utilities.html) and [averager documentation](https://cdat.llnl.gov/documentation/utilities/utilities-1.html) only mention _Spatial Averaging_? If averager can correctly perform a weighted time average over all time steps when the bounds are correctly defined, this should probably be mentioned in the documentation. And the documentation can mention the other functions available for doing more fancy time averaging

- The web documentation and the doc string both mention _giving you control over the order of operations_ but nothing else about that, so there is some ambiguity! Maybe there should be a note that the averaging will be performed left to right (or right to left?) of what is specified in the `axis` parameter

- The [docstring for the `weights` parameter](https://github.com/CDAT/genutil/blob/master/Lib/averager.py#L892) starts with _String specifying weight options_. It should probably more something like _A list of strings specifying weight options, in the same order as the axes specified in the axis parameter_

- The [current averager docststring](https://github.com/CDAT/genutil/blob/master/Lib/averager.py#L860-L995) is not consistent any more with the [web documentation](https://cdat.llnl.gov/documentation/utilities/utilities-1.html)...

**Comment by doutriaux1** [comment]
@jypeter you're right it was moved to genutil a VERY long time ago but left as an import in cdutil for backward compatibilty. The documentation indedd should be in genutil, with a reference to it from cdutil. @reshel3 could you do this? @reshel3 please let me know if you need help with @jypeter other comments. @jypeter I would trust the doc string more, but even these might be out of date 😉

**Comment by jypeter** [comment]
Concerning `averager`, see also https://github.com/CDAT/genutil/issues/32 and https://github.com/CDAT/genutil/issues/33
